catalog/spatial/csw/spatial-csw-endpoint/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/mappings/TestCswRecordMapperFilterVisitor.java

```diff
@@ -488,32 +488,4 @@ public void testLiteralWithNoType() {
         assertThat(literal.getValue(), instanceOf(String.class));
         assertThat(literal.getValue(), is(dateString));
     }
-
```

Comment:
❓ Since we're touching this file, would you mind renaming it to `CswRecordMapperFilterVisitorTest.java` to match our naming convention?